examples/with-turso/app/todo.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+
+import { useTransition } from "react";
+
+import { removeTodo } from "./actions";
+
+export type TodoItem = {
+  id: number;
+  description: string;
+};
+
+export function Todo({ item }: { item: TodoItem }) {
+  const [_, startTransition] = useTransition();
+
+  return (
+    <li className="flex items-center justify-between rounded-md border border-gray-100 p-3">
+      <div className="flex w-full items-center space-x-3">
+        {item.description}
+      </div>
+      <button
+        className="p-1 text-3xl"
+        onClick={() => {
```

Comment:
Should this use an action instead (so you don't need to wrap with a transition?)

Reply:
Updated this to use an action, let me know if that's better.